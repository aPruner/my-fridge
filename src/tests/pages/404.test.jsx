import React from 'react';
import { shallow } from 'enzyme';
import NotFoundPage from '../../pages/404';

test('should render 404 page correctly', () => {
  const wrapper = shallow(<NotFoundPage />);
  expect(wrapper).toMatchSnapshot();
});
