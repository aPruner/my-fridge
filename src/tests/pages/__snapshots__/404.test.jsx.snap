// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render NotFoundPage (404 page) correctly 1`] = `
<Layout>
  <SEO
    title="404: Not found"
  />
  <h1>
    NOT FOUND
  </h1>
  <p>
    You just hit a route that doesn't exist... the sadness.
  </p>
</Layout>
`;
